.github/dependabot.yml

```diff
@@ -1,11 +1,26 @@
 version: 2
 updates:
-- package-ecosystem: pip
-  target-branch: master
+- target-branch: master
+  package-ecosystem: pip
   directory: "/"
   schedule:
     interval: daily
   open-pull-requests-limit: 10
   allow:
     # Allow both direct and indirect updates for all packages
     - dependency-type: "all"
+- target-branch: iso5
+  package-ecosystem: pip
+  directory: "/"
+  schedule:
+    interval: daily
+  open-pull-requests-limit: 10
+  allow:
+    # Allow both direct and indirect updates for all packages
+    - dependency-type: "all"
+  ignore:
+    - dependency-name: "inmanta-core"
+    - dependency-name: "inmanta-license"
+    - dependency-name: "inmanta-lsm"
+    - dependency-name: "inmanta-support"
+    - dependency-name: "inmanta-ui"
```

Comment:
This one is my bad. I only realized now that we actually want to minimize the repos where we apply this on because the difference between iso5 and master means that we would need to diverge branches when we do a major release. As such, we only want to apply this on repos that actually depend on other components. This one doesn't (all other repos I listed do).
```suggestion
```
So I would keep just the change to set up dependabot for iso5, and perhaps update the title and change entry description to match it.

Sorry once more for the miscommunication.

Reply:
No worries :ok_hand: